Makefile

```diff
@@ -63,17 +63,18 @@ start-docker:
 
 # Run dockerized postgres for local development
 postgres:
-	@docker run \
-		--name postgres \
+	@docker ps --format '{{.Names}}' | grep -q indexer-postgres && docker start indexer-postgres || \
+	docker run \
+		--name indexer-postgres \
 		-p 5432:5432 \
-		-e POSTGRES_USER=indexer \
+		-e POSTGRES_USER=rwuser \
```

Comment:
Oh nice! This does both distinguish the user from the DB and brings it closer to deployment usage.